src/view.rs

```diff
@@ -79,7 +79,16 @@ pub fn view_side_menu(m: &Model, side_menu: &SideMenu) -> Html {
 
                     <h3>{"save session"}</h3>
                     <br></br>
-                    <input type="file" onchange=m.link.callback(|value| {
+                    <input type="text" value="filename" onchange=m.link.callback(|v| {
+                        if let ChangeData::Value(s) = v {
+                            return Action::SetSessionTitle(s);
+                        }
+                        Action::Noop
+                    })>
+
+                    </input>
+                    <input type="button" value="Save" onclick=m.link.callback(|_| Action::Noop)>
```

Comment:
change this `Action::Noop` to `Action::SaveSession` even if it isn't implemented yet, so the next implementation step is clear. do the same for the "Save" button in the menu bar.